feature-group-definitions/js-modules-for-service-workers.yml

```
@@ -0,0 +1,7 @@
+name: Service worker JavaScript modules
```

I'm not sure about these names and IDs. It's unclear to me whether the emphasis is on modules (i.e., JS modules for…) or worker types (i.e., Workers accept JS modules).

---

My guess is the devs are asking is "can I use ESM (everywhere)?" more than "I'm writing service worker code, can I use ESM?"

I think we should think about ESM as a single feature that's split into multiple pieces only because of how it's rolled out, and strive to merge it eventually or make a composite feature. For the name I suggest.

```suggestion
name: JavaScript modules in service workers
```

Identifier `js-modules-service-workers` so that we can change our mind about "in" easily.